src/openrct2/ride/water/LogFlume.cpp

```diff
@@ -929,7 +929,7 @@ static void LogFlumeTrack25Down60(
             MetalASupportsPaintSetup(session, kSupportType, MetalSupportPlace::Centre, 8, height, session.SupportColours);
         }
 
-        PaintUtilPushTunnelRotated(session, direction, height, TunnelType::_14);
+        PaintUtilPushTunnelRotated(session, direction, height, TunnelType::SquareFlatTo25Deg);
```

Comment:
StandardFlatTo25Deg

Most likely a copy-paste error when this was copied from the splash boats.